src/index.html

```diff
@@ -17,16 +17,23 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="submit"
+      class="form"
+    >
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input_big"
+      />
```

Comment:
The `data-qa` attribute is used for testing purposes. If you're not using it for testing, it's better to remove it.